addons/account/models/account_move.py

```diff
@@ -219,6 +219,31 @@ def _sequence_year_range_monthly_regex(self):
         related="statement_line_id.statement_id"
     )
 
+    # === Adjusting Entries fields === #
+    adjusting_entry_origin_move_ids = fields.Many2many(
+        comodel_name='account.move',
+        relation='adjusting_entries__account_move',
+        column1='move_id',
+        column2='adjusting_entry_move_id',
+        string="Adjusting Entry Origin Moves",
+    )
+    adjusting_entry_origin_moves_type = fields.Char(compute='_compute_adjusting_entry_origin_moves_type')
```

Comment:
Using this field to store technical labels is a bit too much IMO.

Let's directly use it to put the name to display in the view. Then, we can rename it something like adjusting_entry_origin_label.

You'll still need to compute it. For that, you'll want to compute the label to be used for the move_type of the origin move. You should be able to get it from the field defined on account.move directly. I think, something like dict(self.env['account.move']._fields['move_type'].selection).

Your compute function will need to depend on the active language as well then, with a @api.depends_context('lang').